matrix-meetings-widget/src/components/meetings/MeetingsCalendar/MeetingsCalendarDetailsDialog.test.tsx

```diff
@@ -97,16 +99,46 @@ describe('<MeetingsCalendarDetailsDialog/>', () => {
 
     const dialog = await screen.findByRole('dialog', {
       name: 'An important meeting',
-      description: /^Jan 1, 2999(,| at) 10:00 AM – 2:00 PM$/,
```

Comment:
We discussed before what if and what we want to use as a description for the dialog, and I think this test answers the question: Use the meeting time as the description.